set.mm

```diff
@@ -99022,6 +99022,177 @@ a Cantor normal form (and injectivity, together with coherence
     ABCZADEZFGZAFGZRASHZTUAIABJUBTUAASKLMUASFDEFAFDNOPQ $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Well-Founded Induction
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d A b $.  $d B b c y $.  $d R b c $.  $d R y $.
+    $( Every (possibly proper) subclass of a class ` A ` with a well-founded
+       set-like relation ` R ` has a minimal element.  This is a very strong
+       generalization of ~ tz6.26 and ~ tz7.5 .  (Contributed by Scott Fenton,
+       4-Feb-2011.)  (Revised by Mario Carneiro, 26-Jun-2015.) $)
+    frmin $p |- ( ( ( R Fr A /\ R Se A ) /\ ( B C_ A /\ B =/= (/) ) ) ->
+                   E. y e. B Pred ( R , B , y ) = (/) ) $=
+      ( vb vc wa wss c0 wne cv cpred wceq wrex wi wcel eqeq1d ex cvv syl wfr n0
+      wse frss anim12d wex predeq3 rspcev adantl ctrpred setlikespec trpredpred
+      sess2 ssn0 trpredss jctild adantr trpredex sseq1 neeq1 anbi12d rexeqbi1dv
+      predeq2 imbi12d imbi2d wal dffr4 sylbi vtocl trpredtr imp syl2anc biimprd
+      sp sspred reximdva ssrexv sylsyld sylan9r an31s pm2.61dne exlimdv syl6com
+      syld syl5bi imp32 ) BDUAZBDUCZGZCBHZCIJZCDAKZLZIMZACNZWJWICDUAZCDUCZGZWKW
+      OOWJWGWPWHWQCBDUDCBDUMUEWKEKZCPZEUFWRWOECUBWRWTWOEWRWTWOWRWTGWOCDWSLZIWTX
+      AIMZWOOWRWTXBWOWNXBAWSCWLWSMWMXAICDWLWSUGQUHRUIWTWQWPXAIJZWOOWTWQGZWPGXCC
+      DWSUJZCHZXEIJZGZWOXDXCXHOZWPXDXASPZXICDWSUKZXJXCXGXFXJXAXEHZXCXGOCSDWSULX
+      LXCXGXAXEUNRTCSDWSUOZUPTUQWPXHXEDWLLZIMZAXENZXDWOWPFKZCHZXQIJZGZXQDWLLZIM
+      ZAXQNZOZOWPXHXPOZOFXECDWSURXQXEMZYDYEWPYFXTXHYCXPYFXRXFXSXGXQXECUSXQXEIUT
+      VAYBXOAXQXEYFYAXNIXQXEDWLVCQVBVDVEWPYDFVFYDFACDVGYDFVNVHVIXDXFXPWNAXENWOX
+      DXJXFXKXMTZXDXOWNAXEXDWLXEPZGZWNXOYIWMXNIYIXFWMXEHZWMXNMXDXFYHYGUQXDYHYJC
+      DWSWLVJVKCXEDWLVOVLQVMVPWNAXECVQVRVSWDVTWARWBWEWCWF $.
+  $}
+
+  ${
+    $d A y $.  $d B y $.  $d R y $.
+    $( The principle of well-founded induction.  This principle states that if
+       ` B ` is a subclass of a founded class ` A ` with the property that
+       every element of ` B ` whose initial segment is included in ` A ` is
+       itself equal to ` A ` .  Compare ~ wfi and ~ tfi , which are special
+       cases of this theorem that do not require the axiom of infinity to
+       prove.  (Contributed by Scott Fenton, 6-Feb-2011.)  (Revised by Mario
+       Carneiro, 26-Jun-2015.) $)
+    frind $p |- ( ( ( R Fr A /\ R Se A ) /\ ( B C_ A /\
+                       A. y e. A ( Pred ( R , A , y ) C_ B -> y e. B ) ) ) ->
+                   A = B ) $=
```

Comment:
This and the next statement fit on two lines.

Reply:
Fixed